### ISSUE TITLE
enhance junos_install_config to load from Junos filesys

### ISSUE THREAD

**Issue by jeremyschulman** [Juniper/ansible-junos-stdlib#3]
Add the ability to load configuration stored locally on the Junos filesystem.  Perhaps something like:

```
junos_install_config:
  host={{ inventory_hostname }}
  file=my_backup.conf
  on_host=yes
```

assumes that the file "my_backup.conf" is located in the $HOME directory on the Junos device.

**Comment by shermdog** [comment]
This functionality is not available in PyEZ and thus far hasn't been requested.  Deferring this.

**Comment by stacywsmith** [comment]
This is now possible via the new `juniper_junos_config` module implemented via #290.

The file on the Junos device is specified via the `url` option which allows [any URL format accepted by the device](https://www.juniper.net/documentation/en_US/junos/topics/reference/tag-summary/junos-xml-protocol-load-configuration.html), including a simple file path local to the device.

Given the following config file on the Junos device:
```
user@r0> file show /var/tmp/new_hostname.conf 
system {
  host-name r0.foo;
}
```

This playbook loads the config file:
```
---
- name: "Install from config file on Junos device"
  hosts: junos-all
  connection: local
  gather_facts: no
  roles:
    - Juniper.junos

  tasks:
    - name: "Install from config file on Junos device"
      juniper_junos_config:
        load: "merge"
        url: "/var/tmp/new_hostname.conf"
      register: response
    - name: Print response
      debug:
        var: response
```

Executing the playbook produces the following result:
```
user@h0:~$ ansible-playbook --limit r0 -k ./pb.test.yaml 
SSH password: 

PLAY [Install from config file on Junos device] ************************************************************************

TASK [Install from config file on Junos device] ************************************************************************
changed: [r0]

TASK [Print response] **************************************************************************************************
ok: [r0] => {
    "response": {
        "changed": true, 
        "diff": "\n[edit system]\n-  host-name r0;\n+  host-name r0.foo;\n", 
        "diff_lines": [
            "", 
            "[edit system]", 
            "-  host-name r0;", 
            "+  host-name r0.foo;"
        ], 
        "failed": false, 
        "msg": "Configuration has been: opened, loaded, checked, diffed, committed, closed."
    }
}

PLAY RECAP *************************************************************************************************************
r0                         : ok=2    changed=1    unreachable=0    failed=0   
```
